errbot/backends/irc.py

```diff
@@ -254,7 +267,7 @@ def occupants(self):
         occupants = []
         try:
             for nick in self._bot.conn.channels[self.room].users():
-                occupants.append(IRCMUCOccupant(nick=nick, room=self.room))
+                occupants.append(IRCMUCOccupant(nick, self.room))
```

Comment:
I remove this because in some part (I can't recall now) the keyword argument was incorrect.
Besides this, I don't really think that is "safer" to use kwargs, but i will reset that as they would.